persistent/Database/Persist/Quasi.hs

```diff
@@ -419,838 +419,6 @@ module Database.Persist.Quasi
     , upperCaseSettings
     , lowerCaseSettings
     , nullable
-#if TEST
```

Comment:
All these changes are mostly to support removing the need for this bit of CPP, and also so the test-sutie depends on the library instead of the source files directly.